src/main/java/svc/data/textMessages/SMSAlertDAO.java

```diff
@@ -0,0 +1,164 @@
+package svc.data.textMessages;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.springframework.dao.DataAccessException;
+import org.springframework.dao.IncorrectResultSizeDataAccessException;
+import org.springframework.jdbc.core.PreparedStatementCallback;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.stereotype.Repository;
+
+import svc.data.jdbc.BaseJdbcDao;
+import svc.logging.LogSystem;
+import svc.models.SMSAlert;
+import svc.util.DatabaseUtilities;
+
+@Repository
+public class SMSAlertDAO extends BaseJdbcDao {	
+	private boolean doesAlertExist(String citationNumber, String phoneNumber){
+		Map<String, Object> parameterMap = new HashMap<String, Object>();
+		parameterMap.put("citationNumber", citationNumber);
+		parameterMap.put("phoneNumber", phoneNumber);
+		String sql = "SELECT * FROM sms_alerts WHERE citation_number = :citationNumber AND phone_number = :phoneNumber";
```

Comment:
It would be great if you could put this query in its own sql file like I've started doing in other areas. It keeps the code separate from the sql.